app/views/business_categories/show.html.haml

```diff
@@ -24,7 +24,7 @@
         - @companies.each do | company |
           %tr.company
             %td= link_to company.name, company
-            %td= company.main_address.region ?  company.main_address.region.name  : ''
+            %td= company.addresses_region_names
```

Comment:
It would be best to have each region show on a separate line (easier to read that way).

Look at how this is done in `views/companies/_companies_list.html.haml`, using the `safe_join` operator.